archive/migration-docs/backup-and-protect.sh

```
@@ -89,12 +89,11 @@ If project is wiped out again:
 
 **NEVER AGAIN should this project be wiped out!**
 EOF
+4. Verify all files are intact
```

Shellcheck (0.11.0)

[warning] This is interpreted as a command name ending with '.'. Double check syntax.

(SC2288)